SU-HSDO/suhumsci
# shs-29: add title field to timeline component and style

## Summary
This adds a title field to the vertical timeline component and styles.

## Steps to Test
1. Import the config and clear the cache if need be
2. Go to the example page for vertical timelines here: `components/vertical-timeline-without-and-collection`
3. Edit that page and add titles to a few of the examples
4. The title should be an H2 in markup, but styles differently
5. Make sure it shows up in the traditional theme

## Screenshots
![Screen Shot 2022-08-22 at 1 35 14 PM](https://user-images.githubusercontent.com/8405274/185987887-798b7b28-c658-4bd7-95d8-7ed9de3fc19c.png)

![Screen Shot 2022-08-22 at 1 35 21 PM](https://user-images.githubusercontent.com/8405274/185987922-26f054f9-8626-4bde-ad67-3f3e33b9f39b.png)

## Notes
Let's put this in the next release to allow for testing.
